pom.xml

```diff
@@ -53,6 +53,26 @@
 			<artifactId>spring-security-crypto</artifactId>
 			<version>${security-crypto.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>com.c4-soft.springaddons</groupId>
+			<artifactId>spring-addons-oauth2-test</artifactId>
+			<version>6.1.2</version>
```

Comment:
Just a general note, we should move versions of dependencies to separate properties tag.